test/dash_analytics_test.dart

```diff
@@ -1,10 +1,14 @@
+import 'dart:convert';
```

Comment:
This (and every other file in this repo) needs to have a LICENSE comment. This also needs a top level LICENSE file.